test/Table_Tests/src/Common_Table_Operations/Column_Operations_Spec.enso

```diff
@@ -872,6 +872,45 @@ spec setup =
             t = table_builder [["numbers", [1, 2, 3]]]
             col = t.at "numbers"
             col.text_length . should_fail_with Invalid_Value_Type
+        
+        Test.specify "should handle operation text_left and text_right" <|
+            with_mixed_columns_if_supported [["strings", ["a", "foobar", "", Nothing, "👩‍🔬👩‍🔬V👩‍🔬👩‍🔬", "café", "It was the best of times, it was the worst of times, it was the age of wisdom, it was the age of foolishness, it was the epoch of belief, it was the epoch of incredulity, it was the season of light, it was the season of darkness, it was the spring of hope, it was the winter of despair."]]] t->
+                col = t.at "strings"
+                resLeft = col.text_left 3
+                resLeft.name . should_equal "text_left([strings], 3)"
+                resRight = col.text_right 3
+                resRight.name . should_equal "text_right([strings], 3)"
```

Comment:
Please add tests checking the `value_type` of the returned columns.

I conjecture that with the changes to `GenericBinaryObjectMapOperation`, in the in-memory backends, the columns will become `Mixed`. But they should keep the `Char` type.

Reply:
What is unclear, if I do `r = col.text_left 3`, should the `r.value_type == Value_Type.Char size=3`? Because we do know that this will be the max (and exact) length. I guess there should be no harm in making the type more specific. 

But at the same time, I appreciate, in most use cases we want unbounded strings and I'm wondering if adding this bound could ever cause problem. I guess the only scenario where it's problematic is when the user then uploads such a table to a DB and this creates a fixed-length column when the user did not expect it to be fixed length. I'm not sure how common/problematic that would be though.

Reply:
I've added the tests. But I'm not sure how I can actually restrict the output text size using the APIs that exist today.

Reply:
> But I'm not sure how I can actually restrict the output text size using the APIs that exist today.

Could you please clarify what you mean by 'restrict the output text size'?

Reply:
Right now the output text type is the same width as the input text type. I would like to make the output text width to be the size provided to the left/right function.

Reply:
Ah, I see.

And indeed, the current `GenericBinaryObjectMapOperation` is unable to do that.

However, I think if we provide a custom implementation for `runBinaryMap`, we could create a builder for a fixed-width/max-width text column if the prefix length is constant for the whole column. We'd need to create a separate subclass for that.

Reply:
I am going to leave the output widths as is for now and tackle them as a future ticket. I have some ideas about some more generic changes I would like to make.